eng/Packages.Data.props

```diff
@@ -99,10 +99,10 @@
     <PackageReference Update="System.Security.Cryptography.ProtectedData" Version="4.7.0" />
     <PackageReference Update="System.Threading.Channels" Version="4.7.1" />
     <PackageReference Update="System.Threading.Tasks.Extensions" Version="4.5.4" />
-    <PackageReference Update="System.Text.Json" Version="4.7.2" />
-    <PackageReference Update="System.Text.Encodings.Web" Version="4.7.2" />
+    <PackageReference Update="System.Text.Json" Version="6.0.0" />
```

Comment:
Let's move to the latest version in the 6.x line rather than 6.0.0 for all of these.

Reply:
Updated `System.Text.Json` to `6.0.9`, `6.0.0` is the only stable version for the other two.